.pre-commit-config.yaml

```diff
@@ -52,8 +54,17 @@ repos:
           --diff,
         ]
 
+  - repo: local
```

Comment:
Why is this section necessary? We check that the template is a valid cloudformation template, isn't that enough?

Reply:
cfn-python-lint will check the template semantically, yq will enforce a specific formatting style, like go-fmt. Ok with removing though.